src/screens/Home/components/ReportButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import ButtonBase from '@material-ui/core/ButtonBase';
+import DescriptionIcon from '@material-ui/icons/Description';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+
+interface ReportButtonProps {
+    classes: { root: string; text: string };
+}
+
+const styles = (theme: Theme) =>
+    createStyles({
+        root: {
+            width: '180px',
+            height: '60px',
```

Comment:
We can't set width and height because it'd make the component unresponsive to screen sizes and potentially clip when we translate to burmese